Sources/Clappr_iOS/Classes/Plugin/Playback/AVFoundationPlayback.swift

```diff
@@ -265,19 +268,29 @@ open class AVFoundationPlayback: Playback {
     }
 
     open override func seek(_ timeInterval: TimeInterval) {
+        var timeToSeek = timeInterval
+
+        if supportDVR, let seekStart = seekableTimeRanges.first?.timeRangeValue.start.seconds {
```

Comment:
Why do we need to always get the first timeRangeValue? There is any chance to the second be the right one?

Reply:
Hey @CristianDRM, I changed the code to select the correct time range. Please, take a look.